pkg/flavor/kubernetes/nfs.go

```diff
@@ -23,13 +23,13 @@ import (
 const (
 	nfsPrefix    = "hpe-nfs-"
 	nfsNamespace = "hpe-nfs"
-	nfsImage     = "gcr.io/google_containers/volume-nfs:0.8"
+	nfsImage     = "nimblestorage/nfs-ganesha"
```

Comment:
Do we have a tag for this?

Reply:
sure George, will update the version tag.